test_package/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ set(CMAKE_LIBRARY_OUTPUT_DIRECTORY_RELEASE ${CMAKE_RUNTIME_OUTPUT_DIRECTORY})
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY_DEBUG ${CMAKE_RUNTIME_OUTPUT_DIRECTORY})
 
 # Using the pybind cmake helpers:
-set(PYBIND11_PYTHON_VERSION 2.7)
+set(PYBIND11_PYTHON_VERSION 3)
```

Comment:
should we specify the minor version too?

Reply:
Ubuntu 16.04 has 3.5, Ubuntu 18.04 has 3.6.

It may work if I put 3.5. I don't know if they then ask for a strict version or not...